### PR TITLE
dependency: bump yamlfmt,sys,otelgrpc,otlptracegrpc

### PR DESCRIPTION
**Pull request by sharathsivakumar**
The dependabot created the following PRs:

- https://github.com/etcd-io/etcd/pull/16534
- https://github.com/etcd-io/etcd/pull/16535
- https://github.com/etcd-io/etcd/pull/16536
- https://github.com/etcd-io/etcd/pull/16537
- https://github.com/etcd-io/etcd/pull/16538
- https://github.com/etcd-io/etcd/pull/16539
- https://github.com/etcd-io/etcd/pull/16541
- https://github.com/etcd-io/etcd/pull/16542
- https://github.com/etcd-io/etcd/pull/16543

### CODE REVIEW

**Comment by sharathsivakumar** [comment]
@jmhbnz and @ahrtr All the dependencies have been patched. Please let me know if I need to fix anything.

File: tools/mod/go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module go.etcd.io/etcd/tools/v3
 
-go 1.20
+go 1.21
```
Comment (jmhbnz) [inline tools/mod/go.mod:3]: Please ensure the project stays on go 1.20.  We are not ready to update to go 1.21 and that will be tracked separately when we are.

Comment (sharathsivakumar) [reply tools/mod/go.mod:3]: This was a mistake. Will be fixed soon.

Comment (ahrtr) [reply tools/mod/go.mod]: Please raise a separate PR to bump golang to 1.20.8 firstly. thx @jmhbnz @sharathsivakumar

Comment (ahrtr) [reply tools/mod/go.mod]: It's blocking all other PRs.  Let me bump golang version myself right now.

Comment (ahrtr) [reply tools/mod/go.mod]: FYI. https://github.com/etcd-io/etcd/pull/16554

Comment (ahrtr) [reply tools/mod/go.mod]: https://github.com/etcd-io/etcd/pull/16554 has already been merged, please rebase this PR, thx

**Comment by ahrtr** [comment]
ping @sharathsivakumar to finish this PR. Otherwise there will be another bunch of dependabot PRs next week.

**Comment by sharathsivakumar** [comment]
On it. Will rebase and create the PR.

**Comment by sharathsivakumar** [comment]
done in https://github.com/etcd-io/etcd/pull/16567

**Pull request closed without merging**